### PR TITLE
Add Contact Us page

### PR DESCRIPTION
**Pull request by emiliom**
Follow up from our last Steering Committee meeting

### CODE REVIEW

File: about/contact.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Contact Us
+
+Are you interested in learning more about OceanHackWeek, helping out with the next event, or learning about the next event and how to prepare a strong application? You can:
+
+- Look over this web site, both general pages and [past events](pasthackweeks)
+- Write to us at [oceanhkw@uw.edu](mailto:oceanhkw@uw.edu).
```
Comment (abkfenris) [inline about/contact.md:6]: We could use the info@ cloudflare routed address.

Comment (emiliom) [inline about/contact.md:6]:
```suggestion
- Write to us at [info@oceanhackweek.org](mailto:info@oceanhackweek.org).
```

While we're at it, how about we also change the email address associated with the top-right icons, as part of this PR?

Comment (abkfenris) [reply about/contact.md:6]: 👍

Comment (emiliom) [reply about/contact.md]: I've updated the email address

Comment (emiliom) [reply about/contact.md]: I've updated the email address both here and and in the top-right icon.

**Comment by emiliom** [comment]
I updated the email addresses and there were no other comments. Thanks for your input, @abkfenris .

I'm merging now.

**Pull request merged**